sdk/eventhub/event-hubs/src/partitionPump.ts

```diff
@@ -181,7 +181,10 @@ export function createProcessingSpan(
     }
 
     links.push({
-      context: spanContext
+      context: spanContext,
+      attributes: {
+        enqueuedTime: receivedEvent.enqueuedTimeUtc.getTime()
```

Comment:
Shouldnt the name of the attribute be the same across languages? In which case wouldnt us using camel casing be in conflict with what other languages use?

Reply:
I went with exactly what was posted in the feature request, which was enqueuedTime. I can post back on the issue to ask if casing matters for them (monitoring).

Reply:
Ah! I missed that. We are good